.gitignore

```diff
@@ -138,3 +138,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# VSCode
+.vscode/
```

Comment:
It's usually better to add this to your workstation's global `.gitignore`.
I think the default location is `$HOME/.config/git/` or you can set it with something like `git config --global core.excludesFile ~/.gitignore`